backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/computing/purchasecontract/subtractedvalue/interim/ShipmentNotificationLineLog.java

```diff
@@ -65,4 +69,13 @@ public ProductPrice getPriceActual(@NonNull final IModularContractLogHandler.Cre
 		return super.getPriceActual(request)
 				.negate();
 	}
+
+	@Nullable
+	@Override
+	public ProductPrice getPriceActual(final @NonNull ModularContractLogEntry logEntry)
+	{
+		return Optional.ofNullable(super.getPriceActual(logEntry))
+				.map(ProductPrice::negate)
```

Comment:
why do we have to negate the price?
why did the price come negative?

IMHO if we fix it in this way, I will it is very fragile and will break any time soon.

cc @metas-rc